assets/data/characters/arlecchino/en.json

```diff
@@ -0,0 +1,180 @@
+      "upgrades": [
+        {
+          "name": "Masque of the Red Death Increase",
+          "value": "120.4"
+        },
+        {
+          "name": "1-Hit DMG",
+          "value": "47.5%"
+        },
+        {
+          "name": "2-Hit DMG",
+          "value": "52.11%"
+        },
+        {
+          "name": "3-Hit DMG",
+          "value": "65.39%"
+        },
+        {
+          "name": "4-Hit DMG",
+          "value": "37.15% + 37.15%"
+        },
+        {
+          "name": "5-Hit DMG",
+          "value": "69.98%"
+        },
+        {
+          "name": "6-Hit DMG",
+          "value": "85.38%"
+        },
+        {
+          "name": "Charged Attack DMG",
+          "value": "90.82%"
+        },
+        {
+          "name": "Charged Attack Stamina Cost",
+          "value": "25"
+        },
+        {
+          "name": "High-Speed Movement Stamina Cost",
+          "value": "40/s"
+        },
+        {
+          "name": "Plunge DMG",
+          "value": "63.93%"
+        },
+        {
+          "name": "Low / High Plunge DMG",
+          "value": "127.84% / 159.68%"
+        }
+      ],
+      "type": "NORMAL_ATTACK"
+    },
+    {
+      "name": "All Is Ash",
+      "unlock": "Elemental Skill",
+      "description": "Summons forth Balemoon Bloodfire, dealing Pyro DMG to multiple nearby opponents and performing a dash-cleave against one of them, dealing AoE Pyro DMG.\nOpponents hit by the aforementioned attack will have a Blood-Debt Directive applied to them.\nBlood-Debt Directive\nLasts 30s. Every 5s, it will deal 1 instance of Pyro DMG to the opponent. Max 2 instances. This DMG will be considered Elemental Skill DMG.\nWhen Arlecchino uses a Charged Attack or her Elemental Burst, Balemoon Rising, she will absorb and clear nearby Blood-Debt Directives. Each Directive absorbed grants her a Bond of Life worth 65% of her Max HP.\nThe maximum value of the Bond of Life she can be granted through Blood-Debt Directives within 35s after using her Elemental Skill is 145% of her Max HP. Using the Elemental Skill again during this duration will restart the count on duration and the limit on the value of Bond of Life she may gain from Blood-Debt Directives.",
+      "upgrades": [
+        {
+          "name": "Spike DMG",
+          "value": "14.84%"
+        },
+        {
+          "name": "Cleave DMG",
+          "value": "133.56%"
+        },
+        {
+          "name": "Blood-Debt Directive DMG",
+          "value": "31.8%"
+        },
+        {
+          "name": "CD",
+          "value": "30s"
+        }
+      ],
+      "type": "ELEMENTAL_SKILL"
+    },
+    {
+      "name": "Balemoon Rising",
+      "unlock": "Elemental Burst",
+      "description": "Arlecchino's great wing of Balemoon Bloodfire beats as she absorbs and clears Blood-Debt Directives around her. She deals AoE Pyro DMG before clearing the CD of All Is Ash and healing herself. The healing is based on her Bond of Life value and ATK.",
```

Comment:
Same as above --- text in italic

![image](https://github.com/genshindev/api/assets/82229068/0810a366-f15f-4840-8653-4c16c2d4b958)